src/containers/ListingPage/ListingPage.duck.js

```diff
@@ -330,6 +348,64 @@ export const sendInquiry = (listing, message) => (dispatch, getState, sdk) => {
     });
 };
 
+export const sendTxDetails = (listing, orderData) => async (dispatch, getState, sdk) => {
+  const {
+    bookingDates,
+    selectService: selectedService,
+    location,
+    typeAnimal,
+    rescueDescription,
+  } = orderData;
+  const { bookingStart, bookingEnd } = bookingDates;
+  const protectedData = {
+    selectedService,
+    location,
+    typeAnimal,
+    rescueDescription,
+  };
+  const order = {};
+
+  dispatch(sendTxDetailsRequest());
+  const processAlias =
+    selectedService === ACC_SERVICES.adoption ? txTypes.adoption.alias : txTypes.rescue.alias;
+  const processName =
+    selectedService === ACC_SERVICES.adoption ? txTypes.adoption.process : txTypes.rescue.process;
```

Comment:
Shorten this logic

Reply:
We are trying to access to the same object at the same time, it makes more sense to use spread syntax here